subsystems/art/ShopVFX.py

```diff
@@ -0,0 +1,35 @@
+import pygame
+class ShopVFX:
+    def __init__(self,numTowers, names,numTowers):
+        self.names =  names
+        self.lightenedimages = []
+        self.darkenedimages = []
+        for i in range(0,len(numTowers)):
+            self.lightenedtowers.append(pygame.image.load(".../config/art/lightenedtower%i.jpg"%names[i]))
```

Comment:
this might cause problems, depending on the where this constructor is called . . . something to think about